server/app/services/question/types/QuestionDefinitionConfig.java

```diff
@@ -13,6 +14,7 @@
 
 @AutoValue
 @JsonDeserialize(builder = AutoValue_QuestionDefinitionConfig.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
```

Comment:
This Jackson annotation tells the serializer to only include optional fields if there are not empty. This fixes a bug where null `enumeratorId` fields were being deserialized to 0 and then our code was reading that as if it was an actual enumuratorId.

Reply:
done.